mautic/mautic
# [UI] Remove Merge button when creating a new company

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR solves suggestions from [this stale PR](https://github.com/mautic/mautic/pull/13679/).






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Companies and click on New.
3. In the top-right toolbar, verify that the Merge button is not displayed when creating a new company, but, it is visible once the company is saved, showing the correct modal on clicking it.
